src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3028,86 +3028,15 @@ ProduceBoundNode:
             ' See Section 3 of §11.8.2 Applicable Methods
             ' Deal with Optional arguments. HasDefaultValue is true if the parameter is optional and has a default value.
             Dim defaultConstantValue As ConstantValue = If(param.IsOptional, param.ExplicitDefaultConstantValue(DefaultParametersInProgress), Nothing)
```

Comment:
I considering extracting this section into a separate function [From](https://github.com/dotnet/roslyn/pull/12629/files#diff-fba0994f36e90e71c52cfa15e2acd454L3033) to
[Here](https://github.com/dotnet/roslyn/pull/12629/files#diff-fba0994f36e90e71c52cfa15e2acd454L3151)

Soon I can remove the `GoTo OtherChecks` used in another conditional to call into the section of code.